lib/dialyzer/src/typer.erl

```diff
@@ -187,7 +189,7 @@ version_message() ->
 
 help_message() ->
   S = <<" Usage: typer [--help] [--version] [--plt PLT] [--edoc]
-              [--show | --show-exported | --annotate | --annotate-inc-files]
+              [--show | --show-exported | --annotate | --annotate-inc-files | --anotate-in-place]
```

Comment:
Nitpick:
```suggestion
              [--show | --show-exported | --annotate | --annotate-inc-files | --annotate-in-place]
```

Reply:
Oh gosh, I knew I would make that typo somewhere. Good catch! :)